src/coreComponents/common/format/table/unitTests/testTable.cpp

```diff
@@ -289,41 +248,85 @@ TEST( testTable, layoutTable )
+               "|                               |    local    |    ghost    |                |    local    |    ghost    |    local    |    ghost    |\n"
+               "--------------------------------------------------------------------------------------------------------------------------------------\n"
+               "|             min(local/total)  |          1  |          2  |             3  |          4  |          5  |          6  |          7  |\n"
+               "|             min(local/total)  |          1  |          2  |             3  |          4  |          5  |          6  |          7  |\n"
+               "--------------------------------------------------------------------------------------------------------------------------------------\n\n"
+               );
+  }
+}
+TEST( testTable, testLineWrap )
 {
-  ////////////
-  //////// If setMargin used elsewhere make it public and remove comments for this test
-  ////////////
-  //test with tiny margin
-  // {
-  //   TableLayout tableLayout( {
-  //     TableLayout::ColumnParam{{"Colonne 1"}, TableLayout::Alignment::center},
-  //     TableLayout::ColumnParam{{"Colonne 2"}, TableLayout::Alignment::center},
-  //     TableLayout::ColumnParam{{"Colonne 3"}, TableLayout::Alignment::right},
-  //     TableLayout::ColumnParam{{"Colonne 4"}, TableLayout::Alignment::right},
-  //     TableLayout::ColumnParam{{"Prev\nelement"}, TableLayout::Alignment::right},
-  //     TableLayout::ColumnParam{{"Next\nelement"}, TableLayout::Alignment::right},
-  //   }, "InternalWellGenerator well_injector1" );
-
-  //   //tableLayout.setMargin( TableLayout::MarginValue::tiny );
-
-  //   TableData tableData;
-  //   tableData.addRow( "value 1", "long value 1", "3.0034", 3.0129877, "" , 1 );
-  //   tableData.addRow(  "value 1", "long value 2", "100.45", 4.0129877, 1 , 2 );
-
-  //   TableTextFormatter const tableText( tableLayout );
-  //   EXPECT_EQ( tableText.toString( tableData ),
-// "\n------------------------------------------------------------\n"
-// "|           InternalWellGenerator well_injector1           |\n"
-// "------------------------------------------------------------\n"
-// "|Colonne 1| Colonne 2  |Colonne 3|Colonne 4|   Prev|   Next|\n"
-// "|         |            |         |         |element|element|\n"
-// "------------------------------------------------------------\n"
-// "| value 1 |long value 1|   3.0034|3.0129877|       |      1|\n"
-// "| value 1 |long value 2|   100.45|4.0129877|      1|      2|\n"
-// "------------------------------------------------------------\n\n"
-//              );
-// }
+  TableLayout tableLayout( {"Cras egestas", "CoordX", "C", "CoordZ", "Prev\nelement", "Next\nelement"} );
+  tableLayout.setTitle( "title" ).setMargin( TableLayout::MarginValue::tiny ).disableLineWrap();
+
+  TableData tableData;
+  tableData.addRow( "1", "2", "3.0", 3.0129877, 2.0f, 1 );
+  tableData.addRow( "1", "2", "3.0", 3.0129877, 2.0f, 1 );
```

Comment:
Good thing that we can use a line break in titles :)
- `testLineBreak` would be more accurate as a name (_line wrapping_ term is more dedicated for automatic line breaks).
- I really think it should support line breaks _within content lines_:
```
  tableData.addRow( "1", "2", "one line text", 3.0129877, 2.0f, 1 );
  tableData.addRow( "1", "2", "three\nlines\ntext", 3.0129877, 2.0f, 1 );
  tableData.addRow( "1", "2", "one line text", 3.0129877, 2.0f, 1 );
```

Reply:
I can make line break for cell values if needed, not sure if right this is mandatory

Reply:
I encountered a scenario that needed this, but I forgot it :(
We can get back on this later.